run-code-coverage.sh

```diff
@@ -3,6 +3,12 @@
 set -x
 set -e
 
-CARGO_INCREMENTAL=0 RUSTFLAGS='-Cinstrument-coverage' LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw' cargo test --features compile-blueprints-at-build-time --no-fail-fast
```

Comment:
Out of interest, we seem to have removed `compile-blueprints-at-build-time` - does it still work? I guess so :D.
Perhaps we just miss some coverage around the scrypto compiler, but we can ignore that.

Reply:
I ran into some issue with pre-build.